### DISCUSSION TITLE
Is there a risk where automatically created daily notes being overwritten?

### DISCUSSION THREAD

**Discussion by Liu233w** [remotely-save/remotely-save#451] [Q&A] [answered]
Since there is no conflict resolution, I can imagine a scenario where the daily notes are overwritten:
1. When the user opens ob on device A the first time on a day, a daily note is created automatically.
2. The user enter something into the note, syncing it to the cloud
3. User opens ob on device B (before syncing is triggered), a daily note is created automatically
4. Syncing is triggered. Since the daily note on device B is created after the note on device A, the note on the cloud is overwritten with blank daily note on device B

Can this really happen?

**Comment by fyears** [comment, marked as answer]
yes. in the near future ( hopefully a month) a new release will have conflict management
